_data/email.yml

```diff
@@ -121,6 +123,7 @@ websites:
     - name: T-Online
       url: https://email.t-online.de
```

Comment:
That URL redirects to the site without HTTPS. Should this be changed in this PR or a separate one?